Clinical-Genomics/cgweb
# Disallow spaces in container name

This PR fixes the problem that customers sometimes get leading or trailing spaces in the container name which LIMS chokes at 

How to test leading/trailing spaces in order name:
1. intstall on stage on clinical db 
1. login to op stage
1. create order with leading and/or trailing spaces in the order name
1. save

Expected outcome:
Red box around ordername indicating bad ordername

How to test leading/trailing spaces in sample name:
1. intstall on stage on clinical db 
1. login to op stage
1. create microbial order with leading and/or trailing spaces in the container name
1. save sample

Expected outcome:
Red box around containername indicating bad containername

How to test leading/trailing spaces in family-sample name:
1. intstall on stage on clinical db 
1. login to op stage
1. create MIP order with leading and/or trailing spaces in the container name
1. save sample

Expected outcome:
Red box around containername indicating bad containername
